packages/ui-react-utils/src/getElementType.ts

```diff
@@ -25,6 +25,16 @@
 import { logWarn as warn } from '@instructure/console'
 import { ComponentType, ReactElement } from 'react'
 
+type AsElementType = keyof JSX.IntrinsicElements | ReactElement | ComponentType
```

Comment:
I think we will have to move this type to the `shared-types` package once it is merged to master.

Reply:
@Brailor I just did, can you please recheck it?